apps/explorer/lib/explorer/repo.ex

```diff
@@ -153,5 +153,18 @@ defmodule Explorer.Repo do
       otp_app: :explorer,
       adapter: Ecto.Adapters.Postgres,
       read_only: true
+
+    alias Explorer.Repo, as: ExplorerRepo
+
+    def init(_, opts) do
+      extra_postgres_parameters = [application_name: ExplorerRepo.get_application_name() <> "-replica1"]
```

Comment:
What's `replica1`?

Reply:
it's the name of the repo defined in this module, added upstream that is intended to be connected to a read replica.

https://github.com/celo-org/blockscout/pull/746/files/6c4c7a59f50d4169b191293c11e3206eeb0aa0ca#diff-4b8ee0e9de96fe0baf675b2d78397e1862b9a61d46a2d7b8ea6c3d67b7190031R151

Reply:
Gotcha.